lib/compat/wordpress-6.5/navigation-block-variations.php

```diff
@@ -11,7 +11,7 @@ function gutenberg_navigation_link_variations_compat( $args ) {
 	if ( 'core/navigation-link' !== $args['name'] || ! empty( $args['variation_callback'] ) ) {
 		return $args;
 	}
-	$args['variation_callback'] = 'build_navigation_link_block_variations';
+	$args['variation_callback'] = 'gutenberg_block_core_navigation_link_build_variations';
```

Comment:
Should there really be the `gutenberg_` prefix here? I only see a function called `block_core_navigation_link_build_variations`, without the prefix. Here:

https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/navigation-link/index.php#L354

Reply:
It's my understading that the code in the lib/compat folder only gets loaded if the gutenberg plugin is active on a wordpress installtion with version 6.4/6.3. But the code is not merged into core. And the gutenber plugin prefixes all functions in the block-library with 'gutenberg_' on build process, to avoid a clash with existinf core functions. 
In my local testing with wp 6.4 this works and the function is called -> the variations for core post types and post types registeres before init#10 are created.

Reply:
You're right, I didn't know this 🙂